devops/charts/bg-monitoring/templates/influxdb/deployment.yaml

```diff
@@ -0,0 +1,77 @@
+{{ if .Values.influxdb.enabled }}
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  annotations:
+{{- with .Values.influxdb.deploymentAnnotations }}
```

Comment:
above annotations?

Reply:
no.It gets all annotations from env.yaml or values yaml if they exist